README.md

```diff
@@ -201,14 +204,14 @@ the database to create, etc.
 ## Requirements
 
 * Docker
-* [Minikube v0.25+](https://github.com/kubernetes/minikube/releases/tag/v0.25.0)
-* [Helm v2.8.2+](https://helm.sh)
+* [Minikube](https://github.com/kubernetes/minikube/releases/)
```

Comment:
If we're not pointing to a specific release, we might as well point to their main page instead.
```suggestion
* [Minikube](https://minikube.sigs.k8s.io/)
```